modin/pandas/base.py

```diff
@@ -1250,27 +1250,18 @@ def ffill(self, axis=None, inplace=False, limit=None, downcast=None):
 
     pad = ffill
 
-    def fillna(
+    @_inherit_docstrings(pandas.DataFrame.fillna)
```

Comment:
I think we should inherit docstrings for DataFrame and Series in dataframe.py and series.py, respectively, because here not only function name is different and new parameters have been added.

Reply:
I had to add this inheritance to satisfy our doc_checker script. It fails if no docstring is found here.

Reply:
maybe it's better to just write a new docstring?..

Reply:
Ok I took a docstring from pandas and updated for our rules.